gdal/ogr/ogrsf_frmts/ngw/drv_ngw.html

```diff
@@ -0,0 +1,217 @@
+
+<h2>Geometry</h2>
+
+NextGIS Web supports only one geometry column. Default spatial reference is Web
+Mercator (EPSG:3857). The following geometry types are available:
+
+<ul>
+    <li>POINT
+    <li>LINESTRING
+    <li>POLYGON
+    <li>MULTIPOINT
+    <li>MULTILINESTRING
+    <li>MULTIPOLYGON
+</ul>
+
+<h2>Field data types</h2>
+
+NextWeb supports only following field types:
+
+<ul>
+    <li>OFTInteger
+    <li>OFTInteger64
+    <li>OFTReal
+    <li>OFTString
+    <li>OFTDate
+    <li>OFTTime
+    <li>OFTDateTime
+</ul>
+
+<h2>Paging</h2>
+
+Features are retrieved from the server by chunks if server support this feature.
+This number can be altered with the NGW_PAGE_SIZE configuration option or
+PAGE_SIZE open option.
+
+<h2>Write support</h2>
+
+Datasource and layers creation and deletion is possible.
+Write support is only enabled when the datasource is opened in update mode and
+user have permissions.
+
+Vector and PostGIS layers insert and update operations may cached if BATCH_MODE
+is enable. Delete operation executes immediately.
+
+<h2>Open options</h2>
+
+Starting with GDAL 2.0, the following open options are available:
+
+<ul>
+    <li>USERPWD - Username and password, separated by colon.
+    <li>PAGE_SIZE=-1 - Limit feature count while fetching from server. Default
+        value is -1 - no limit.
+    <li>BATCH_SIZE=-1 - Size of feature insert and update operations cache before send to server. If batch size is -1 batch mode is disable. Default value is -1.
+</ul>
+
+<h2>Dataset creation options</h2>
+
+The following dataset/datasource creation options are available:
+
+<ul>
+    <li>KEY - Key value. Must be unique in whole NextGIS Web instance. Optional.
+    <li>DESCRIPTION - Resource description. Optional.
+    <li>USERPWD - Username and password, separated by colon.
+    <li>PAGE_SIZE=-1 - Limit feature count while fetching from server. Default
+        value is -1 - no limit.
+    <li>BATCH_SIZE=-1 - Size of feature insert and update operations cache before send to server. If batch size is -1 batch mode is disable. Default value is -1.
+</ul>
+
+<h2>Layer creation options</h2>
+
+The following layer creation options are available:
+
+<ul>
+    <li>OVERWRITE - Whether to overwrite an existing table with the layer name
+        to be created. The resource will delete and new one will created. This
+        leads that resource identifier will change. Defaults to NO. Optional.
+    <li>KEY - Key value. Must be unique in whole NextGIS Web instance. Optional.
+    <li>DESCRIPTION - Resource description. Optional.
+</ul>
+
+<h2>Metadata</h2>
+
+NextGIS Web metadata are supported in datasource, vector, PostGIS, raster layers
+and styles. Metadata stored at specific domain "NGW". NextGIS Web supported
+metadata as strings and numbers. Metadata keys with decimal numbers will have
+suffix <b>.d</b> and for real numbers - <b>.f</b>. To create new metadata item,
+add new key=value pair in NGW domain using <i>SetMetadataItem</i> function and
```

Comment:
use the

Reply:
Fixed per 4579aeab927d70ddab6142340bf5550c6789b42e